File: pset6/mario.py
from cs50 import get_int
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 23:
        break
steps = 1
spaces = height - 1

for i in range(height):
    print(' ' * spaces, end = "")
    print('#' * steps, end = " ")
    print('#' * steps, end = "")
    print(' ' * spaces, end = "\n")
    spaces-=1
    steps+=1

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main()
{
    int height, spacing, stars;
    do
    {
        printf("Height: ");
        height = get_int();
    }while (height > 23 || height <0);
    spacing = height - 1;
    stars = 1;
    
    
    for(int pheight = 0; pheight < height; pheight++)
    {
        //prints spacing
        for(int i = 0; i < spacing; i++)
        {
            putchar(' ');
        }
        //prints first row
        for(int i = 0; i < stars; i++)
        {
            putchar('#');
        }
        printf(" ");
        //prints second row
        for(int i = 0; i < stars; i++)
        {
            putchar('#');
        }
        stars++;
        spacing--;
        putchar('\n');
    }

}